{
	"id": 172479285,
	"body": "Fuller trace:\r\n\r\n```\r\nruntime: memory allocated by OS (1626533888) not in usable range [0x9143a000,0x1143a000)\r\nruntime: out of memory: cannot allocate 268435456-byte block (3145728 in use)\r\nfatal error: out of memory\r\nruntime stack:\r\n runtime.throw(0xa23d7a60, 0xd)\r\n    /usr/local/go/src/runtime/panic.go:530 +0x80\r\n runtime.largeAlloc(0x10000000, 0x1, 0x25)\r\n    /usr/local/go/src/runtime/malloc.go:764 +0xe8\r\nruntime.mallocgc.func3()\r\n    /usr/local/go/src/runtime/malloc.go:660 +0x34\r\nruntime.systemstack(0xbed29de4)\r\n    /usr/local/go/src/runtime/asm_arm.s:243 +0x8c\r\nruntime.mstart()\r\n    /usr/local/go/src/runtime/proc.go:1042\r\ngoroutine 17 [running, locked to thread]:\r\nruntime.systemstack_switch()\r\n    /usr/local/go/src/runtime/asm_arm.s:188 +0x4 fp=0x91476cb8 sp=0x91476cb4\r\nruntime.mallocgc(0x10000000, 0xa221c4c0, 0x1, 0x4)\r\n    /usr/local/go/src/runtime/malloc.go:661 +0xb04 fp=0x91476d28 sp=0x91476cb8\r\nruntime.newarray(0xa221c4c0, 0x4000000, 0x91658000)\r\n    /usr/local/go/src/runtime/malloc.go:794 +0xec fp=0x91476d48 sp=0x91476d28\r\nruntime.makeslice(0xa2210118, 0x4000000, 0x0, 0x4000000, 0x0, 0x0, 0x0, 0x0)\r\n```",
	"user": {
		"login": "karalabe",
		"id": 129561,
		"type": "User",
		"site_admin": false
	},
	"created_at": "2016-01-18T09:43:30Z",
	"updated_at": "2016-01-18T09:43:30Z"
}
